src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[];
+  tempTodo: Todo | null;
+  onDelete: (id: number) => void;
+  updateTodo: (todo: Todo) => Promise<void>;
+};
+
+export const TodoList: React.FC<Props> = React.memo(({
+  todos,
+  tempTodo,
+  onDelete,
+  updateTodo,
+}) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings